src/main/java/com/redhat/devtools/lsp4ij/features/semanticTokens/LSPSemanticTokenTypes.java

```diff
@@ -0,0 +1,85 @@
+/*******************************************************************************
+ * Copyright (c) 2025 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution,
+ * and is available at http://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ * Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+
+package com.redhat.devtools.lsp4ij.features.semanticTokens;
+
+import org.eclipse.lsp4j.SemanticTokenTypes;
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+/**
+ * Custom semantic token types.
+ */
+@ApiStatus.Internal
+public class LSPSemanticTokenTypes {
+
+    // https://code.visualstudio.com/api/language-extensions/semantic-highlight-guide#standard-token-types-and-modifiers
+    private static final LSPSemanticTokenType Label = new LSPSemanticTokenType(
```

Comment:
This encapsulates the notion of a label and removes the hard-coded text attributes association from `DefaultSemanticTokensColorProvider`.